client/src/components/Ticketing/ticketingmanager/AdminPurchase/AdminPurchase.tsx

```diff
@@ -478,14 +492,38 @@ const AdminPurchase = () => {
       headerName: '',
       width: 150,
       renderCell: (params) => (
-        <button
-          className='p-2 rounded-lg text-zinc-500 hover:text-red-600 hover:bg-red-100
-            focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-500'
-          onClick={() => removeRow(params.row.id)}
-          aria-label='Delete ticket'
-        >
-          <TrashCanIcon className='h-5 w-5' strokeWidth={2} />
-        </button>
+        !params.row.complimentary ? (
+          <button
+            className='bg-red-500 px-2 py-1 text-white rounded-xl hover:bg-red-600 disabled:opacity-40 m-2'
+            onClick={() => removeRow(params.row.id)}
+          >
+            Remove
+          </button>
```

Comment:
It looks like you may have accidentally removed a recent change when fixing the merge conflicts. Do you think you could replace the "Remove" button with this icon button that currently exists on `main`.
```
<button
  className='p-2 rounded-lg text-zinc-500 hover:text-red-600 hover:bg-red-100
  focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-500'
  onClick={() => removeRow(params.row.id)}
  aria-label='Delete ticket'
>
  <TrashCanIcon className='h-5 w-5' strokeWidth={2} />
</button>
```